elisa/src/org/labkey/elisa/actions/ElisaUploadWizardAction.java

```diff
@@ -111,24 +112,34 @@ public boolean executeStep(ElisaRunUploadForm form, BindException errors) throws
                         errors.addError(new ObjectError("main", null, null, e.toString()));
                     }
                 }
-                return false;
+
+                // if this is a manual metadata upload, continue to the concentrations input view
+                if (form.getSampleMetadataInputFormat() == SampleMetadataInputFormat.MANUAL)
+                    return false;
+                else
+                    return super.executeStep(form, errors);
             }
 
             @Override
             public ModelAndView getNextStep(ElisaRunUploadForm form, BindException errors) throws ServletException, SQLException, ExperimentException
             {
                 if (form.isResetDefaultValues() || errors.hasErrors())
                     return getRunPropertiesView(form, !form.isResetDefaultValues(), false, errors);
-                else
+                else if (form.getSampleMetadataInputFormat() == SampleMetadataInputFormat.MANUAL)
                     return getConcentrationsView(form, false, errors);
-            }
+                else
+                    return null;            }
         };
     }
 
     @Override
     protected void addRunActionButtons(ElisaRunUploadForm form, InsertView insertView, ButtonBar bbar)
     {
-        addNextButton(bbar);
+        // don't render the concentrations view
+        if (form.getSampleMetadataInputFormat() == SampleMetadataInputFormat.MANUAL)
+            addNextButton(bbar);
+        else
+            addFinishButtons(form, insertView, bbar);

```

Comment:
Likely not part of this PR changes, but I noticed in the UI that the final step of the ELISA assay wizard for the "Concentrations for Standard Wells", the button bar doesn't have the btn-primary styling for the "Save and Finish", etc. buttons.